src/main/kotlin/com/openlattice/hazelcast/HazelcastQueue.kt

```diff
@@ -53,15 +53,27 @@ class HazelcastQueue<T> internal constructor(val name: String) : TypedQueueIdent
         private val instanceChecker = UniqueInstanceManager(HazelcastQueue::class.java)
 
         // When adding new entries to this list, please make sure to keep it sorted and keep the name in sync
-        @JvmField val AUDITING = HazelcastQueue<AuditableEvent>("AUDITING")
-        @JvmField val BACKGROUND_INDEXING  = HazelcastQueue<EntitySet>("BACKGROUND_INDEXING")
-        @JvmField val EMAIL_SPOOL  = HazelcastQueue<RenderableEmailRequest>("EMAIL_SPOOL")
-        @JvmField val ID_GENERATION  = HazelcastQueue<UUID>("ID_GENERATION")
-        @JvmField val INDEXING = HazelcastQueue<UUID>("INDEXING")
-        @JvmField val QUEUED_INTEGRATION_JOBS = HazelcastQueue<QueuedIntegrationJob>("QUEUED_INTEGRATION_JOBS")
-        @JvmField val LINKING_CANDIDATES = HazelcastQueue<EntityDataKey>("LINKING_CANDIDATES")
-        @JvmField val LINKING_INDEXING = HazelcastQueue<Triple<List<Array<UUID>>, UUID, OffsetDateTime>>("LINKING_INDEXING")
-        @JvmField val LINKING_UNINDEXING = HazelcastQueue<Triple<List<Array<UUID>>, UUID, OffsetDateTime>>("LINKING_UNINDEXING")
-        @JvmField val TWILIO = HazelcastQueue<MessageRequest>("TWILIO")
+        @JvmField
```

Comment:
I know it's a bit non-standard but the @JvmField annotation on the same line makes it a lot easier to read IMHO. I'd like to keep it. There was previous discussion on the HazelcastMap PR.

Reply:
I agree, but the autoformatter converts it to this, so I feel like we should either update the autoformatter or leave it this way. I'm happy to change it back by hand but I feel like this will be an ongoing pain

Reply:
Vote for update autoformatter

Reply:
ticket = made
https://jira.openlattice.com/browse/LATTICE-2394

Reply:
Alright I manually reset it and disabled the autoformatter.

In general, I don't love disabling the autoformatter for specific cases since we'll lose *all* autoformatting which leaves room for other bad style patterns without manual upkeep and I tend to prefer consistency anyway, although I think it's fine in this case since HazelcastQueue is pretty simple and shouldn't be updated that often, and I left a TODO to re-enable once the autoformatter is updated.